enclave_mgmt/compile_models.py

```diff
@@ -375,10 +375,12 @@ def collect_moodle_dfs(bucket, prefix):
     }
 
 
```

Comment:
I believe this PR is currently missing three things:

1. It should include pydantic models for the two new objects (see examples [here](https://github.com/openstax/raise-enclave/blob/12c268e4c49ab277d40fa7946f22f0b75bcb32d6/enclave_mgmt/compile_models.py#L107-L124)).
2. In [create_models](https://github.com/openstax/raise-enclave/blob/12c268e4c49ab277d40fa7946f22f0b75bcb32d6/enclave_mgmt/compile_models.py#L248), it should call a method for each new df similar to [this invocation of question_contents_model](https://github.com/openstax/raise-enclave/blob/12c268e4c49ab277d40fa7946f22f0b75bcb32d6/enclave_mgmt/compile_models.py#L137) that uses the pydantic model to validate the schema
3. It should write the final CSV files out [here](https://github.com/openstax/raise-enclave/blob/12c268e4c49ab277d40fa7946f22f0b75bcb32d6/enclave_mgmt/compile_models.py#L260-L275) after the schema check. Otherwise they won't actually end up in the `CSV_OUTPUT_DIR` / enclave environment.